pandas/indexes/base.py

```diff
@@ -723,7 +723,8 @@ def copy(self, name=None, deep=False, dtype=None, **kwargs):
             new_index = new_index.astype(dtype)
         return new_index
 
-    __copy__ = copy
+    def __copy__(self, **kwargs):
+        return self.copy(**kwargs)
```

Comment:
`__deepcopy__` already exists in this class, see [line 1483](https://github.com/pandas-dev/pandas/blob/master/pandas/indexes/base.py#L1483).  Did you want me to move it?